### PR TITLE
Add linchpin image

### PR DESCRIPTION
**Pull request by joejstuart**
This also add boto (python interfaces to aws) for python3, since it's already there for python2.

### CODE REVIEW

File: linchpin/ansible.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[defaults]
```
Comment (arilivigni) [inline linchpin/ansible.cfg:1]: Do we want these too?
inventory = /ansible/inventory
ansible_force_color = true

Comment (joejstuart) [reply linchpin/ansible.cfg:1]: Will the inventory property change any functionality? I just copied the config from the sample project.

Comment (arilivigni) [reply linchpin/ansible.cfg:1]: That is where it is as well for ansible if we don't need it for linchpin that is fine they should probably be consistent
https://github.com/CentOS-PaaS-SIG/contra-env-sample-project/blob/master/config/dockerfiles/ansible/ansible.cfg

**Review by arilivigni — APPROVED** [review]
LGTM

File: linchpin/ansible.cfg (hunk 1 above)
Comment (joejstuart) [reply linchpin/ansible.cfg:1]: Cool. I added both of those.

File: linchpin/ansible.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+inventory = /ansible/inventory
+ansible_force_color = true
```
Comment (arilivigni) [inline linchpin/ansible.cfg:3]: Not sure if this had to be True vs true

**Pull request merged**